test-requirements.txt

```diff
@@ -0,0 +1,2 @@
+charm-tools
```

Comment:
Not a huge fan of using the PyPI package over the snap, but it's not that big of a deal.

Reply:
I run into build time issues with using the snap version due to what appears to be inconsistency between python version in the snap and on the system the test is running on.

The issue goes away when I build it from pypi at test run time.  I'm happy to try to solve the build time issue I see if you prefer to stick with charm tools from snap.

Reply:
The system Python *ought* to have no effect on the build process, since it should only use components bundled within the snap.  I suspect, though, that this has more to do with the issue mentioned above that we're still trying to compile some artifacts when we're really only intending to cache them.  Using the pypi package is fine for now until we can get a better solution.